### DISCUSSION TITLE
Docker Swarm image updates

### DISCUSSION THREAD

**Discussion by awptechnologies** [containrrr/watchtower#1793] [Q&A]
Will watchtower work for updating docker swarm images. 

If so how do I deploy it? Do I run it on single manager node or do I replicate it across all manager nodes?

**Comment by piksel** [comment]
I don't think it's possible to have watchtower and swarm play nicely.

[shepherd](https://github.com/containrrr/shepherd) could be a better match since it's built for swarm from the start.
